cmd/main.go

```diff
@@ -76,10 +62,9 @@ func main() {
 	manifest.AppType = cfg.App.Type
 	mattermost.AddRoutes(r, &manifest, staticAssets, cfg.App.Secret, cfg.Debug)
 
-	// Gameday Routes
```

Comment:
also I think that this should stay here because if there is config there is no need for configure at all. So if `DB` is set there is no need for `/configure` routes